whatToDad_site/whatToDad_app/templates/index.html

```diff
@@ -1,101 +1,100 @@
 {% extends "base.html" %}
 {% load static %}
 
- {% block content %}
+{% block content %}
 
 <style>
-    
-    body {
-        font-family: "Roboto", sans-serif;
-        font-size: 18px;
-        background-color: #fdfdfd;
-    }
+  body {
+    font-family: "Roboto", sans-serif;
+    font-size: 18px;
+    background-color: #fdfdfd;
+  }
 
-    h3 {
-        margin-right: 10px;
-    }
+  h3 {
+    margin-right: 10px;
+  }
 
-    img {
-      height: 20em;
-      width: 50em;
-    }
+  img {
+    height: 20em;
+    width: 50em;
+  }
 
-    .head_text{
-      color: white;
-    }
-    .card{
-      box-shadow: 0 16px 48px #E3E7EB;
-    }
+  .head_text {
+    color: white;
+  }
 
-    .rating {
-      display: flex;
-      justify-content: flex-end;
-      align-items: center;
-    }
+  .card {
+    box-shadow: 0 16px 48px #E3E7EB;
+  }
 
-    .rating > input{ display:none;}
+  .rating {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
 
-    .rating > label {
-      position: relative;
-      width: 1em;
-      font-size: 3vw;
-      color: #FFD600;
-      cursor: pointer;
-    }
+  .rating>input {
+    display: none;
+  }
 
-    .rating > label::before{ 
-      content: "\2605";
-      font-size: 3vw;
-      position: absolute;
-      opacity: 0;
-    }
+  .rating>label {
+    position: relative;
+    width: 1em;
+    font-size: 3vw;
+    color: #FFD600;
+    cursor: pointer;
+  }
 
-    .rating > label:hover:before,
-    .rating > label:hover ~ label:before {
-      opacity: 1 !important;
-    }
+  .rating>label::before {
+    content: "\2605";
+    font-size: 3vw;
+    position: absolute;
+    opacity: 0;
```

Comment:
Opacity is probably not the right thing to use here. The content is still selectable/copyable. Could you change the color/background-color/background-image instead?

Reply:
I went ahead and removed the opacity, as I don't believe we actually need it.